packages/blade/src/components/Dropdown/_decisions/decisions.md

```diff
@@ -0,0 +1,672 @@
+# Dropdown & ActionList
+
+<br/>
+
+**Top-View Skeleton**
+
+```jsx
+<Dropdown>
+  <Dropdown.SelectInput label="Item Selection" />
```

Comment:
what happens if we don't provide `<Dropdown.SelectInput/>` and directly provide `<Dropdown.Overlay/>`?

Reply:
We should throw an error ideally.